svc-systems/systems/getresource.go

```diff
@@ -645,6 +645,17 @@ func (p *PluginContact) GetSystemResource(ctx context.Context, req *systemsproto
 		return resp
 
 	}
+
+	urlInfo := req.URL
+	res := strings.Split(urlInfo, "/")
+	if res[5] == "BootOptions" && len(res) == 7 {
```

Comment:
just leave a comment here for the check using hardcode

Reply:
Comment resolved